xml/System.ServiceModel.Syndication/SyndicationFeed.xml

```diff
@@ -1924,9 +1929,30 @@
         <ReturnType>System.Collections.ObjectModel.Collection&lt;System.String&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a <see cref="T:System.Collections.ObjectModel.Collection" /> of strings indicating the set of values in the 'skipDays' element for the feed.</summary>
```

Comment:
This cref is causing a build warning. We can either use natural language to describe it (the preferred way) or we can get the cref fixed. My suggestion is the former:

```suggestion
        <summary>Gets a collection of strings indicating the set of values in the 'skipDays' element for the feed.</summary>
```